base/reflection.jl

```diff
@@ -538,31 +538,6 @@ struct type with no fields.
 """
 issingletontype(@nospecialize(t)) = (@_pure_meta; isa(t, DataType) && isdefined(t, :instance))
 
-"""
```

Comment:
Why does this change necessitate removing this function entirely?

Reply:
The function's only here so that it could be exported from Compat in the past. As the docstring said, don't use this.